### PR TITLE
Updated all req.user.userID references

### LINKED ISSUE #66: Update references to req.user.userID in groupsControllers file
Currently the groupsControllers are references req.user.userID to get the current user's ID, but with the passport.js update, the new variable is req.user._id.

### PR DESCRIPTION
**Pull request by marice-romero**
### Attached Issue
This PR resolves #66 

## Change Summary
With the update to auth, the current user's ID is now stored in `req.user._id`. The groupsController file was previously referencing `req.user.userID` to get the current user's ID. Please note that this will need to be merged with the #47 branch that implements passport.js.

Also realized the code to test whether the current user is an owner or member of the group was not working in the `getGroup` function, so that was removed for now (will fix that later).

## I Had Difficulty With
It was an easy fix, for once.

### CODE REVIEW

**Review by msrezaie — APPROVED** [review]
Simple fix. Gotta update/merge this after the passportjs PR is merged into dev, but I am sure you already knew that.

**Review by angelkbrown — APPROVED** [review]
Thanks for noticing this and making this quick change, @marice-romero! I tested it with Mo's branch and it's working well for the create and list endpoints! Nice!! 🎉 💯

**Pull request merged**
